roles/test_operator/README.md

```diff
@@ -68,7 +80,7 @@ Execute tests via the [test-operator](https://openstack-k8s-operators.github.io/
       externalPlugin: "{{ cifmw_test_operator_tempest_external_plugin | default([]) }}"
       extraRPMs: "{{ cifmw_test_operator_tempest_extra_rpms | default([]) }}"
       extraImages: "{{ cifmw_test_operator_tempest_extra_images | default([]) }}"
-    tempestconfRun: "{{ cifmw_tempest_tempestconf_config | default(omit) }}"
+    tempestconfRun: "{{ cifmw_test_operator_tempest_tempestconf_config | default(omit) }}"
```

Comment:
I didn't update the docs with all the last changes because I'm still working and testing some of them, will fix as soon as I'll finish with that. regarding the job impact this should be solved in the corresponding patch we discussed.

Reply:
Ok, thanks for the explanation. I'm keeping this open so that we do not forget about this.